GetFed/GetFed/Helpers/Identity.swift

```diff
@@ -35,6 +36,8 @@ enum Identity: String {
         switch self {
         case .homeToFoodSearchSegue:
             return "homeToFoodSearch"
+        case .foodSearchToFoodDetailSegue:
```

Comment:
Again, you should probably just be using the `rawValue` for these identifiers.

Reply:
I guess I'm still unclear about the _disadvantage_ to this vs using the rawValue. With the way I set it up, the syntax itself calls out what the values are (.segueID, .cellID, etc.).